### DISCUSSION TITLE
My student devloper pack

### DISCUSSION THREAD

**Discussion by Ammu123467** [community/community#22493] [New to GitHub]
<p>Hi,</p>
<p>I have applied for the Student developer pack on the 15th February 2022, the status is still pending. I don’t have the school-issued email id, so applied with the school document, with the valid date written on it. Today is the 24th February they said I will receive an email from them too me if they reject it or approve it pls just tell me it is rejected or accepted</p>

**Comment by mpboom** [comment, marked as answer]
<p>Yes, you will receive an email upon either approval or rejection. The approval process however, can take anywhere between a few days and multiple weeks. Please have some patience while your request is being processed. On the forum, we cannot assist or dicuss these requests.</p>
